Examples/UIExplorer/ListViewExample.js

```diff
@@ -57,17 +57,20 @@ var ListViewSimpleExample = React.createClass({
           dataSource={this.state.dataSource}
           renderRow={this._renderRow}
           renderScrollComponent={props => <RecyclerViewBackedScrollView {...props} />}
-          renderSeparator={(sectionID, rowID) => <View key={`${sectionID}-${rowID}`} style={styles.separator} />}
+          renderSeparator={this._renderSeperator}
         />
       </UIExplorerPage>
     );
   },
 
-  _renderRow: function(rowData: string, sectionID: number, rowID: number) {
+  _renderRow: function(rowData: string, sectionID: number, rowID: number, highlightRow: funct) {
```

Comment:
@rops thanks, no doubt that's possible... but it's all still a mystery at the moment since I've not yet dug into flow. I've updated the argument per your direction and that seems to have done the trick. Thanks!